script/CallPause.s.sol

```diff
@@ -17,11 +17,11 @@ contract CallPause is MultisigBuilder {
         IMulticall3.Call3[] memory calls = new IMulticall3.Call3[](1);
 
         calls[0] = IMulticall3.Call3({
-            target: _optimismPortalAddr(),
+            target: _superchainConfigAddr(),
             allowFailure: false,
             callData: abi.encodeCall(
                 Pausable.pause,
-                ()
+                ("presigner")
```

Comment:
@zchn do you have an opinion as to whether we should keep this string? We can remove it from the `DeputyGuardianModule`'s interface.